articles/quickstart.md

```diff
@@ -9,22 +9,29 @@ uid: microsoft.quantum.write-program
 ---
 # Write your first quantum program
 
-Learn how to write a quantum program using the Microsoft QDK. You will write an application to demonstrate quantum entanglement, which is called a **Bell state**.
+In this QuickStart, we show you how to write your first Q# program that manipulates and measures qubits and demonstrates the effects of superposition and entanglement.  This guides you on installing the QDK, building the program and executing that program on a quantum simulator.  
 
-You start with the simplest program possible and build it up to demonstrate [quantum superposition and quantum entanglement](xref:microsoft.quantum.overview.what#the-qubit).
+You will write an application called Bell to demonstrate quantum entanglement.  The name Bell is in reference to Bell states, which are specific quantum states of two qubits that are used to represent the simplest examples of superposition and quantum entanglement. 
 
 ## Pre-requisites
 
+If you are ready to start coding, follow these steps before proceeding: 
+
 * [Install](xref:microsoft.quantum.install) the Quantum Development Kit using your preferred language and development environment
 * If you already have the QDK installed, make sure you have [updated](xref:microsoft.quantum.update) to the latest version
 
-## What You'll Learn
+You can also follow along with the narrative without installing the QDK, reviewing the overviews of the Q# programming language and the first concepts of quantum computing.
+
+## Demonstrating qubit behavior with Q#
+
+Recall our simple [definition of a qubit](xref:microsoft.quantum.overview.what#the-qubit).  Where classical bits hold a single binary value such as a 0 or 1, the state of a qubit can be in a **superposition** of 0 and 1 simultaneously.  Conceptually, a qubit can be thought of as a direction in space (also known as a vector).  A qubit can be in any of the possible directions. The two **classical states** are the two directions; representing 100% chance of measuring 0 and 100% chance of measuring 1.  This representation is also more formally visualized by the [block sphere]
```

Comment:
```suggestion
Recall our simple [definition of a qubit](xref:microsoft.quantum.overview.what#the-qubit).  Where classical bits hold a single binary value such as a 0 or 1, the state of a qubit can be in a **superposition** of 0 and 1.  Conceptually, a qubit can be thought of as a direction in space (also known as a vector).  A qubit can be in any of the possible directions. The two **classical states** are the two directions; representing 100% chance of measuring 0 and 100% chance of measuring 1.  This representation is also more formally visualized by the [block sphere]
```

Reply:
```suggestion
Recall our simple [definition of a qubit](xref:microsoft.quantum.overview.what#the-qubit).  Where classical bits hold a single binary value such as a 0 or 1, the state of a qubit can be in a **superposition** of 0 and 1 simultaneously.  Conceptually, a qubit can be thought of as a direction in space (also known as a vector).  A qubit can be in any of the possible directions. The two **classical states** are the two directions; representing 100% chance of measuring 0 and 100% chance of measuring 1.  This representation is also more formally visualized by the [Bloch sphere]
```

Reply:
For the first, I would like to keep "simultaneously", as noted above.  I don't see what the change you suggest in the second, but I did correct the "block sphere" link